Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -530,7 +530,7 @@ class BrowserViewController: UIViewController {
         if PLCrashReporter.shared().hasPendingCrashReport() {
             PLCrashReporter.shared().purgePendingCrashReport()
             showRestoreTabsAlert()
-        } else {
+        } else if !AppConstants.IsRunningTest {
```

Comment:
we do. I didnt add this. This was initially in the `restoretabs()` in TabManager section but I didn't want that behavior there as the test would never restore tabs. So I moved it up into BVC in order to keep the behaviour for tests that need it.